contracts/eosio.system/include/eosio.system/native.hpp

```diff
@@ -76,63 +124,94 @@ namespace eosiosystem {
       EOSLIB_SERIALIZE( abi_hash, (owner)(hash) )
    };
 
-   /*
-    * Method parameters commented out to prevent generation of code that parses input data.
+   // Method parameters commented out to prevent generation of code that parses input data.
+   /**
+    * The EOSIO core native contract that governs authorization and contracts' abi.
+    * 
+    * @details
     */
    class [[eosio::contract("eosio.system")]] native : public eosio::contract {
       public:
 
          using eosio::contract::contract;
 
          /**
-          *  Called after a new account is created. This code enforces resource-limits rules
-          *  for new accounts as well as new account naming conventions.
+          * New account action 
+          * 
+          * @details Called after a new account is created. This code enforces resource-limits rules
+          * for new accounts as well as new account naming conventions.
           *
-          *  1. accounts cannot contain '.' symbols which forces all acccounts to be 12
-          *  characters long without '.' until a future account auction process is implemented
-          *  which prevents name squatting.
+          * 1. accounts cannot contain '.' symbols which forces all acccounts to be 12
+          * characters long without '.' until a future account auction process is implemented
+          * which prevents name squatting.
           *
-          *  2. new accounts must stake a minimal number of tokens (as set in system parameters)
-          *     therefore, this method will execute an inline buyram from receiver for newacnt in
-          *     an amount equal to the current new account creation fee.
+          * 2. new accounts must stake a minimal number of tokens (as set in system parameters)
+          * therefore, this method will execute an inline buyram from receiver for newacnt in
+          * an amount equal to the current new account creation fee.
           */
          [[eosio::action]]
          void newaccount( name             creator,
                           name             name,
                           ignore<authority> owner,
                           ignore<authority> active);
 
-
+         /**
+          * Update authorization action.
+          * TO DO: Ovi annotate the last 8 actions
```

Comment:
Any reason this can't be done in the current PR?

Reply:
ah, leftover.